### DISCUSSION TITLE
Agenda compiti

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#396] [Idee e proposte]
Buongiorno, sarebbe possibile ,nella sezione agenda del docente ,nella fase in cui il docente assegna i compiti per casa ,aggiungere un opzione che includa la condivisione dello stesso compito a più classi ,senza dover fare copia e incolla classe per classe? Un po' come si fa per le circolari/avvisi.

**Comment by trinko** [comment]
La segno come richiesta.
